membership-attribute-service/app/actions/WithBackendFromCookieAction.scala

```diff
@@ -4,12 +4,9 @@ import components.TouchpointBackends
 import configuration.Config
 import play.api.mvc.{ActionRefiner, Request, Result}
 import services.IdentityAuthService
-
 import scala.concurrent.{ExecutionContext, Future}
-import play.api.libs.concurrent.Execution.Implicits.defaultContext
-
 
-class WithBackendFromCookieAction(touchpointBackends: TouchpointBackends, ex: ExecutionContext) extends ActionRefiner[Request, BackendRequest] {
+class WithBackendFromCookieAction(touchpointBackends: TouchpointBackends)(implicit ex: ExecutionContext) extends ActionRefiner[Request, BackendRequest] {
```

Comment:
didn't comment last time because I didn't know, but @davidfurey said he made everything explicit for Dep injection rather than a mixture, although not a massive issue